Anatoly Shlom/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Paris</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/style.css">
+</head>
+<body>
+    <main>
+        <article class="wdata">
+            <section class="w_date">
+                <div class="cur_week_day">
+                    Friday
+                </div>
+                <div class="cur_date">
+                    August, 23rd
+                </div>
+            </section>
+            <section class="info">
+                <div class="temp">
+                    12&#176;
+                </div> 
+                <div class="loc">
+                    Paris, France
+                </div>
+                <div class="wind">
+                    <div class="wind_img">
+                        <img src="img/Wind.png" alt="Wind">
```

Comment:
It's better to use `img` tag for content, for example for images of any post, or for user's avatar. When image is just a decorate element it's better to add it via CSS to avoid reading that by robots as a part of meaningful content of the page.

Reply:
Hmmm... It seems to me, that in that widget this icons are part of the content. I have no text alternatives to this weather icons according to mockup. So if user blocks images loading in his device - using my solution he will see "alt" content(also it helps screen-readers as i understand), and if i put them to div - he wouldn't get any information about weather conditions at all. But i can be wrong)))

Reply:
That's strong opinion! I suppose one day you will be ready for debating it on the production project 👍